### DISCUSSION TITLE
Send custum checks like nagios

### DISCUSSION THREAD

**Discussion by simo222** [TwiN/gatus#686] [Ideas]
Hi

Is it possible to create custom check on any system.  using bash or python and send a row, column, color, duration to the server. Like nagios/xymon.

For example by triggering an api which does a post and show on dashboard. If client has sent yellow color, it will show yellow. If he send green it will show green. If red it will show red.

**Comment by ser** [comment]
it's already deployed into gatus, check documentation

**Reply by simo222** [reply]
Hi.  Which section?

**Reply by TwiN** [reply]
Look at external endpoints.

That said, right now, you only get to pick between green (success) and red (failure).

**Reply by ser** [reply]
We hope for more, though! 🥇

**Reply by simo222** [reply]
thanks. awesome. yellow color would be perfect
